tests/main/views/test_direct_award.py

```diff
@@ -326,3 +327,38 @@ def test_search_api_urls_convert_to_correct_frontend_urls(self, search_api_url,
         doc = html.fromstring(body)
         assert html_escape(frontend_url) in body
         assert len(doc.xpath('//a[@href="{}"]'.format(frontend_url)))
+
+
+class TestDirectAwardEndSearch(TestDirectAwardBase):
+    def setup_method(self, method):
+        super(TestDirectAwardEndSearch, self).setup_method(method)
+
+        self.g9_search_results = self._get_g9_search_results_fixture_data()
```

Comment:
This is already in the base class isn't it. (Looks like you copied from `TestDirectAwardURLGeneration`, but that doesn't inherit from `TestDirectAwardBase`.) Same goes for some other line in the setup method I think?